src/dialogs/AccountAuthorizationDialog.vue

```diff
@@ -20,18 +20,19 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import Vue from 'vue'
+import { Component, Prop, Emit } from 'vue-property-decorator'
 
 @Component({})
 export default class AccountAuthorizationDialog extends Vue {
-  // Prop to display the dialog.
-  @Prop() readonly dialog: boolean
+  /** Prop to display the dialog. */
```

Comment:
This is a JSDoc style comment (displayed on hover of the prop). The old code is a regular comment and doesn't display on hover.